autotest/test_mf6.py

```diff
@@ -1579,6 +1579,23 @@ def test_multi_model(function_tmpdir):
     sim.write_simulation()
     sim.run_simulation()
 
+    try:
+        # test kwargs error checking
+        wel = ModflowGwfwel(
+            gwf2,
+            print_input=True,
+            print_flows=True,
+            stress_period_data=welspd,
+            save_flows=False,
+            auxiliary="CONCENTRATION",
+            pname="WEL-1",
+            param_does_not_exist=True,
+        )
+        ex = False
+    except flopy.mf6.mfbase.FlopyException:
+        ex = True
+    assert ex
```

Comment:
The usual way to test exceptions is with [`pytest.raises()`](https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions); see [here](https://github.com/modflowpy/flopy/blob/1e6991d7c05f97b6b9464edecfd49101dd7969a2/autotest/test_grid.py#L129-L130) for an example in this repo.

Reply:
Thanks for the comment.  I updated the code to use pytest.raises().